GITHUB_APP_IMPLEMENTATION_PLAN.md

```
@@ -396,21 +345,32 @@ Located at `github-app-manifest.json`. Key settings:
 
 ### 🚧 Remaining Work
 
-**Improved Subscription UX (Priority 1):**
+**Priority: OAuth Token Renewal**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)